kaizen/cli/cli.py

```
@@ -1,21 +1,151 @@
+        for field in required_fields:
+            if field not in config:
+                raise click.ClickException(f"Missing required field '{field}' in config")
+        
+        # Support both old 'tests' format and new 'steps' format
+        if 'tests' not in config and 'steps' not in config:
+            raise click.ClickException("Config must contain either 'tests' or 'steps' section")
+            
+        return config
+        
+    except yaml.YAMLError as e:
+        raise click.ClickException(f"Invalid YAML in config file: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)